deploy/charts/cert-manager/values.yaml

```diff
@@ -14,6 +14,9 @@ global:
   rbac:
     create: true
 
+  podSecurityPolicy:
+    enabled: false
```

Comment:
:+1: we can look at enabling this by default in a future release once we have more feedback